modules/dedicated-accessing-your-cluster.adoc

```diff
@@ -7,8 +7,29 @@
 
 To access your {product-title} cluster:
 
-. Log in to link:https://cloud.redhat.com/[cloud.redhat.com].
+. Log in to link:https://cloud.redhat.com/openshift[cloud.redhat.com/openshift].
 
-. Select *Create Cluster* -> *Red Hat-Managed Cluster*.
+. Select *Create Cluster* -> *Red Hat OpenShift Dedicated*.
 
-. Enter your cloud provider credentials.
+. Enter your *Cluster name*, number of *Compute nodes*, and select an *AWS Region*.
+
+. Select your *Node Type*. The number and types of nodes available to you depend
+upon your {product-title} subscription.
+
+. If you wish to configure your networking IP ranges un *Advanced Options*, the
```

Comment:
seems to be missing a few words:

> ...networking IP ranges un *Advanced Options*...

What does this mean? Is this under Advanced Options?

Reply:
Yep, good catch!